lib/output.js

```diff
@@ -134,7 +135,10 @@ function OutputStream(options) {
 
     function make_string(str, quote) {
         var dq = 0, sq = 0;
-        str = str.replace(/[\\\b\f\n\r\v\t\x22\x27\u2028\u2029\0\ufeff]/g,
+        var re = options.xml_safe ?
+            /[\x00-\x1f\x22\x27\u2028\u2029\0\ud800-\udfff\ufeff\ufffe\uffff]/g :
+            /[\\\b\f\n\r\v\t\x22\x27\u2028\u2029\0\ufeff]/g;
+        str = str.replace(re,
           function(s, i){
```

Comment:
I appreciate this code was formatted as such previsouly, but could you please combine lines 141 and 142 as follows:
```
        str = str.replace(re, function(s, i){
```
This would make the indentation of the closing brace of the function make more sense.